### PR TITLE
Test MapServer with MAP

### PR DESCRIPTION
**Pull request by sbrunner**
Fix: GEORD-279

### CODE REVIEW

File: acceptance_tests/docker-compose.yaml [hunk 1]
```diff
@@ -8,8 +8,22 @@ services:
...
+    volumes:
+      - ./config/mapserver/:/etc/mapserver
+    user: www-data
+
+  mapserver-map:
+    image: camptocamp/mapserver:${DOCKER_TAG}
+    environment:
+      MS_DEBUGLEVEL: 5
+      MAPSERVER_CATCH_SEGV: 1
+      MS_MAPFILE:
```
Comment (danduk82) [inline acceptance_tests/docker-compose.yaml:22]: to test if `MS_MAPFILE` is used, you should put the mapfile somewhere else and set this variable to the mapfile

File: acceptance_tests/docker-compose.yaml [hunk 2]
```diff
@@ -8,8 +8,22 @@ services:
...
+    links:
+      - db
+    #ports:
+    #  - 8380:8080
     volumes:
       - ./config/mapserver/:/etc/mapserver
```
Comment (danduk82) [inline acceptance_tests/docker-compose.yaml:28]: change this location

**Review by danduk82 — CHANGES_REQUESTED** [review]
either I don't understand how it works, or the mapfile for `mapserver-map` should be put somewhere else than `/etc/mapserver/` and `MS_MAPFILE` should be set to point to that mapfile

File: acceptance_tests/docker-compose.yaml (hunk 1 above)
Comment (sbrunner) [reply acceptance_tests/docker-compose.yaml:22]: ? I want to test the MAP argument...

File: acceptance_tests/docker-compose.yaml (hunk 2 above)
Comment (sbrunner) [reply acceptance_tests/docker-compose.yaml:28]: If I do that I can't test any more if the default MS_MAP_PATTERN is working...

**Comment by sbrunner** [comment]
The goal is to test if the map parameter is working with the default configuration (especially the default MS_MAP_PATTERN)...

**Comment by danduk82** [comment]
Ah ok, now I get it

**Review by danduk82 — APPROVED** [review]

**Pull request merged**